### ISSUE TITLE
sf org list : does not render properly in terminal if you have an expired org

### ISSUE THREAD

**Issue by curtisk** [forcedotcom/cli#2533]
### Summary
When using the `sf org list` command since upgrading to 2.13.9 npm version, the output no longer works properly when there are expired previously authorized (non-scratch) orgs in list

### Steps To Reproduce
Run `sf org list` from a terminal/command prompt, or within the terminal in vscode, when you have a previously authorized org credentials expire

### Expected result
The org list should render just as it normally does

### Actual result
The output has intermixed markup code including a large block of svg related markup on the rows that have expired creds. Instead of having the colored words there, the svg markup bleeds into the output, in my case on the rows with markup one row was 13719 characters wide, blowing out the output render

### System Information
cmd.exe (both standalone and with vscode Terminal, git bash on windows (same)

```json
{
  "architecture": "win32-x64",
  "cliVersion": "@salesforce/cli/2.13.9",
  "nodeVersion": "node-v18.16.0",
  "osVersion": "Windows_NT 10.0.22621",
  "rootPath": "C:\\Users\\curtisk\\AppData\\Roaming\\npm\\node_modules\\@salesforce\\cli",
  "shell": "cmd.exe",
  "pluginVersions": [
    "@oclif/plugin-autocomplete 2.3.9 (core)",
    "@oclif/plugin-commands 2.2.28 (core)",
    "@oclif/plugin-help 5.2.20 (core)",
    "@oclif/plugin-not-found 2.4.3 (core)",
    "@oclif/plugin-plugins 3.9.1 (core)",
    "@oclif/plugin-search 0.0.23 (core)",
    "@oclif/plugin-update 3.2.4 (core)",
    "@oclif/plugin-version 2.0.1 (core)",
    "@oclif/plugin-warn-if-update-available 2.1.1 (core)",
    "@oclif/plugin-which 2.2.34 (core)",
    "@salesforce/cli 2.13.9 (core)",
    "apex 2.3.19 (core)",
    "auth 2.8.20 (core)",
    "community 2.3.15 (user)",
    "data 2.5.16 (core)",
    "deploy-retrieve 1.19.0 (core)",
    "info 2.6.49 (core)",
    "limits 2.3.38 (core)",
    "login 1.2.35 (core)",
    "marketplace 0.3.0 (core)",
    "org 2.10.12 (core)",
    "packaging 1.24.0 (user)",
    "schema 2.3.29 (core)",
    "settings 1.4.33 (core)",
    "sobject 0.2.11 (core)",
    "source 2.10.40 (core)",
    "telemetry 2.3.6 (core)",
    "templates 55.5.14 (core)",
    "trust 2.6.18 (core)",
    "user 2.3.36 (core)",
    "@salesforce/sfdx-scanner 3.16.0 (user)"
  ]
}
```
### Additional information
@salesforce/cli/2.13.9 win32-x64 node-v18.16.0
2 screenshots attached, I piped the output to txt file to make sense of what was going on, since the in terminal view was a massive scrollfest. **If I run `sf org list --skipconnectionstatus` that renders fine since it omits the Status column details.**

<img width="999" alt="sf-org-list-bug-1" src="https://github.com/forcedotcom/cli/assets/2028068/c4d8b4b7-8f0c-4a5f-b338-584b36bbda12">
<img width="792" alt="sf-org-list-bug-2" src="https://github.com/forcedotcom/cli/assets/2028068/b220db8b-221a-4065-970a-1fadbaefe9a6">

[sf-bug-version.json](https://github.com/forcedotcom/cli/files/13064907/sf-bug-version.json)

**Comment by curtisk** [comment]
added version details json ^

**Comment by cristiand391** [comment]
@curtisk thanks for reporting!
we got a fix for this, you can try it by installing sf from the nightly channel:
https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_update_cli.htm#sfdx_setup_install_cli_rc

or wait until it's gets promoted to RC in 1 week (and to stable the week after that).

**Comment by curtisk** [comment]
Great news, and quick turnaround, thanks!

**Comment by jshackell-sfdc** [comment]
This issue is fixed in [2.15.9 (Nov 1, 2023)](https://github.com/forcedotcom/cli/blob/main/releasenotes/README.md#2159-nov-1-2023-stable).
